blueprints/networking/f5-lb/variables.tf

```diff
@@ -0,0 +1,208 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "disk_size" {
+  type        = number
+  description = "The default disk size in GB for F5 VMs."
+  default     = 100
+}
+
+variable "forwarding_rules_config" {
```

Comment:
here too I would not expose the module-level variable, let's just do what makes sense here